app/src/main/kotlin/org/eclipse/kuksa/companion/feature/door/surface/DoorVehicleSceneThread.kt

```diff
@@ -101,7 +100,11 @@ class DoorVehicleSceneThread(
         doorTrunkRearProperty?.set(doorTrunkRear.value.toFloat)
     }
 
-    fun moveCamera(yawValue: Float, pitchValue: Float, camDistanceValue: Float) {
+    fun moveCamera(
```

Comment:
I guess this is now the default :)

https://pinterest.github.io/ktlint/1.0.1/rules/standard/#function-signature